renovate.json

```diff
@@ -3,6 +3,11 @@
   "extends": [
     "config:recommended"
   ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "automerge": true,
```

Comment:
I wonder if Renovate can get around the PR approval requirement (it probably cannot)?

Reply:
Good point. What do you think of trying and changing it if it does not work?

Reply:
Sure